packages/lwc-language-server/src/__tests__/typing-indexer.test.ts

```diff
@@ -66,6 +76,11 @@ describe('TypingIndexer', () => {
 <CustomLabels xmlns="http://soap.sforce.com/2006/04/metadata"/>
 `;
             jest.spyOn(fsExtra, 'readFileSync').mockReturnValue(Buffer.from(xmlDocument));
+            jest.spyOn(typingIndexer, 'customLabelFiles', 'get')
+                .mockReturnValue([
+                    '/foo/bar/test-workspaces/sfdx-workspace/utils/meta/labels/CustomLabels.labels-meta.xml'
+                ]);
```

Comment:
Under the hood, multiple `readFileSync` invocations occurred. First in `sfdxConfig` (indirectly invoked by `customLabelFiles`) and then again in `saveCustomLabelTypings`. Because of this, I'm surprised the test actually passed before.

Making this change seems to preserve the intent of the test, and provides the unit-level isolation such that only a single `readFileSync` invocation occurs.